### PR TITLE
HTML

### PR DESCRIPTION
**Pull request by Epresh**
the first 2 submissions i did were very identical and i was corrected by KENNY on my indentation and i have corrected it with some new features that makes the form entirely diffrent from what i submitted before.

### CODE REVIEW

**Review by goke-epapa — COMMENTED** [review]
Good progress, a few comments:

1. It looks like you have multiple versions of the same file, please remove older ones and add only the latest file to the repository.

2. Please remove the space and dots in your filenames, you can review this style guide: https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md
Suggestion for filename: `pagent_ng.html`

**Comment by Epresh** [comment]
Thank you very much.
I would do that right away.

**Review by Rakshith-V — COMMENTED** [review]
I think you're good to go if you can indent it properly. You can use extensions like 'prettify' if you are using visual studio code.

**Review by Rakshith-V — DISMISSED** [review]
You can go ahead and merge after you remove redundant files as said and indent the code. If possible, also comment when necessary.

**Comment by tegaphilip** [comment]
@Epresh I have just checked the changes you made, and it looks like you still haven't added the indentations that were suggested. You also haven't renamed your html file. As @goke-epapa suggested, `pagent_ng.html` is a much better name than `PAgent.ng..html`. Rename the file appropriately and indent your code. Thank you.

**Comment by Epresh** [comment]
Okay i wud do that this evening.

**Review by adaugochi — DISMISSED** [review]

File: submissions/PreciousAisosaErhabor/HTML/PAgent_ng.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+ 
+<title>https://PaGent.ng/registration</title>
+  <img scr="PaGent.ng.png" height="10"; width="10"/>
```
Comment (tegaphilip) [inline submissions/PreciousAisosaErhabor/HTML/PAgent_ng.html]: @Epresh `scr` should be `src`. Please commit this image file `PaGent.ng.png` to github as well and push your changes. Remove the `;` after the height attribute value. Test your code on a browser to confirm that it works as expected.

**Review by tegaphilip — DISMISSED** [review]

**Review by Gbensky — APPROVED** [review]

**Review by tegaphilip — APPROVED** [review]

**Pull request merged**